clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleServerGroup.groovy

```diff
@@ -113,6 +115,37 @@ class GoogleServerGroup {
           null
     }
 
+    /**
+     * @return The load balancing policy containing the
+     * capacity metrics and named ports this server group is configured with for all L7 backends.
+     */
+    GoogleHttpLoadBalancingPolicy getLoadBalancingPolicy() {
+      def loadBalancingPolicy = null
+      def loadBalancers = GoogleServerGroup.this.loadBalancers
+      loadBalancers.each { GoogleLoadBalancerView view ->
+        if (GoogleLoadBalancerType.valueOf(view.loadBalancerType) == GoogleLoadBalancerType.HTTP) {
+          GoogleHttpLoadBalancer.View httpView = view as GoogleHttpLoadBalancer.View
+          def backendServices = Utils.getBackendServicesFromHttpLoadBalancerView(httpView)
+          backendServices.each { GoogleBackendService backendService ->
+            def foundBackends = backendService.backends.findAll { GoogleLoadBalancedBackend backend ->
+              GCEUtil.getLocalName(backend.serverGroupUrl) == GoogleServerGroup.this.name
+            }
+            // Note: we configure the same load balancing policy for all backends on a create/clone,
```

Comment:
This is intended to to be used as the suggestion in the server group wizard for load balancing policy. If a server group outside is created and added to L7 outside of Spinnaker, this will return the first load balancing policy we find. _Maybe_ we could count the backends and report the most common configuration, but it seems like overkill at this point, since you can configure this anyway.